src/GovUk.Education.ExploreEducationStatistics.Common/Model/FileInfo.cs

```diff
@@ -12,6 +12,8 @@ public class FileInfo
         public string FileName { get; set; }
         public string Name { get; set; }
         public string Size { get; set; }
+        public string UserName { get; set; }
+        public DateTime? Created { get; set; }
```

Comment:
Have you considered the need to set these fields in other places that FileInfo is created, for example in `FileExtension#ToFileInfoNotFound` should we be setting some default/blank values? How about places like where we return image files, methodology image files, and chart files that all use `FileInfo`? This might be another case for introducing other view models here, or we might just not care.